src/ts/options.ts

```diff
@@ -72,6 +78,17 @@ $(function () {
     getOptions();
 });
 
+export function parseOptions (options: string) {
+    return Object.assign({}, defaultOptions, JSON.parse(options))
+}
+
+export function parseEditorOptions (options: string) {
```

Comment:
The editor complained about unsupported options if we do not create an editor option object from a whitelisted list of options keys it accepts.